libraries/rush-lib/src/logic/operations/ProjectLogWritable.ts

```diff
@@ -216,10 +216,8 @@ export async function initializeProjectLogFilesAsync(
     jsonl: jsonlPath
   } = logFilePaths;
   await Promise.all([
-    FileSystem.ensureFolderAsync(logFolderPath),
+    FileSystem.ensureEmptyFolderAsync(logFolderPath),
```

Comment:
This is problematic. All logs for a project share the same folder, even across phases.